arbitrum-docs/how-arbitrum-works/fraud-proofs/wavm-floats.md

```diff
@@ -6,7 +6,7 @@ would be an extremely tricky endeavor.
 WASM specifies floating point operations as being compliant to IEEE 754-2019,
 which is not deterministic, and full of edge cases.
 
-Instead, floating point operations (apart from trivial bit-casts like i32 <-> f32)
+Instead, floating point operations (apart from trivial bit-casts like i32 \<-\> f32)
```

Comment:
[Context](https://docusaurus.io/blog/preparing-your-site-for-docusaurus-v3#bad-usage-of--1)